k2/csrc/aux_labels.h

```diff
@@ -46,6 +46,9 @@ struct AuxLabels {
   std::vector<int32_t> labels;
 };
```

Comment:
Note: when we integrate the Array stuff, we would have `using AuxLabels = Array2<int32_t, int32_t>` or something like that.
Does one of you have time to take what I sketched out, take into account you guys' comments (you found some small issues) and start switching stuff over?

Reply:
... just using Array for AuxLabels, and not for the other stuff, may be a good way to start that is not too overwhelming.